src/components/main-layout/ContentPanelTimer.tsx

```diff
@@ -42,7 +42,7 @@ const ContentPanelTimer: React.FC<ContentPanelProps> = ({
             return;
         }
         const count: number = 1;
-        const FADE_TIME = 100;
+        const FADE_TIME = 150;
```

Comment:
Nit: Might be nice to merge the two instances of this value (or add a comment that it must be synced with CSS)